Pipfile

```diff
@@ -16,6 +16,8 @@ dataclasses = "*"
 importlib_metadata = "*"
 requests = "*"
 docopt = "*"
+uvicorn = "*"
```

Comment:
Are we going to be using both?

Reply:
For a while we will keep both until we make a choice.